docs/src/content/docs/reference/frontmatter.md

```diff
@@ -166,6 +169,7 @@ interface HeroConfig {
     link: string;
     variant: 'primary' | 'secondary' | 'minimal';
     icon: string;
+    attrs?: HTMLAttributes<"a">;
```

Comment:
Let’s use an explicit type here as `HTMLAttributes` is an Astro type which we don’t document anywhere:
```suggestion
    attrs?: Record<string, string | number | boolean>;
```

Reply:
If I undo the changes to the docs it will not help with the translation tracker problem because there are commits modifying this files, or am I wrong?